src/pages/login/login.ts

```diff
@@ -0,0 +1,31 @@
+import template from './login.hbs';
+import Block from '../../utils/Block';
+import { Mainlogo } from '../../components/mainlogo';
+import { AuthRow, AuthProps } from '../../components/auth_row';
+import { Button } from '../../components/button';
+
+export const authrows: Array<object> = [ {div_class: "", name: "login", type: "text", placeholder: "Login"}, 
+                            {div_class: "", name: "password", type: "password", placeholder: "Password"} ]
```

Comment:
1. Точно стоит поправить форматирование. Сейчас сложно воспринимать информацию
2. Почему object? это не сильно отличается от any. К тому же есть тип AuthProps
```suggestion
export const authrows: Array<AuthProps> = [ {div_class: "", name: "login", type: "text", placeholder: "Login"}, 
                            {div_class: "", name: "password", type: "password", placeholder: "Password"} ]
```